docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -26,11 +27,9 @@ do the following:
 - Click on the hamburger menu, click Modify 
 - Choose the workloads listed above and click Modify
 
-During the last few weeks of a development cycle for a quarterly release (versions 15.3, 15.6, etc.), it is possible for the Roslyn codebase to make use of new language features, which will not be suppored by the last released version. During that period, it is recommended to use the [preview version of Visual Studio](https://www.visualstudio.com/vs/preview/) or [Roslyn nightlies](https://github.com/dotnet/roslyn/issues/18783#issuecomment-299064434). Alternatively, you can just ignore some red squiggles produced by the IDE that do not correspond to build errors (on portions of the code using the latest language features).
```

Comment:
I am deleting this for many reasons:

1. "Quarterly release" and the stated version numbers were both wrong and also not concepts that are meaningful to non-Microsoft users.
2. Saying "hey, this might be broken, let's send you down this path of sadness" isn't a good way to say we might depend on it. If we have this dependency problem, it should be clearly called out in machine setup above.
3. I'm not sure the statement was correct anyways.